sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -224,7 +225,7 @@ public String getId() {
                                    item,
                                    requestOptions,
                                    true)
-                   .map(response -> ModelBridgeInternal.createCosmosAsyncItemResponse(response, itemType))
+                   .map(response -> ModelBridgeInternal.createCosmosAsyncItemResponse(response, itemType, getItemDeserializerWithoutDecryption()))
```

Comment:
Why decryption is excluded?

Reply:
initially I didn't want to decrypt the encrypted fields on write. so for that reason decryption was excluded for write operation.

However this is changed now to be in par with .Net. addressed.